build.gradle.kts

```diff
@@ -38,6 +38,7 @@ kotlin {
                 implementation(libs.kotest.framework.engine)
                 implementation(libs.kotest.assertions.core)
                 implementation(libs.kotest.assertions.table)
+                implementation(libs.kotlinx.datetime)
```

Comment:
issue: if this library is used only in tests we should use `testImplementation` configuration instead